From mingo@elte.hu  Fri May 16 12:22:28 2008
From: Ingo Molnar <mingo@elte.hu>
Date: Fri, 16 May 2008 09:30:14 +0200
Subject: USB: build fix
To: Greg Kroah-Hartman <gregkh@suse.de>
Message-ID: <20080516073014.GA18443@elte.hu>
Content-Disposition: inline

From: Ingo Molnar <mingo@elte.hu>

this config:

http://redhat.com/~mingo/misc/config-Wed_Apr_30_15_12_48_CEST_2008.bad

fails to build due to an #error. Turn that into a #warning instead
to not break randconfig builds unnecessarily.

Signed-off-by: Ingo Molnar <mingo@elte.hu>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/net/usb/cdc_subset.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/usb/cdc_subset.c
+++ b/drivers/net/usb/cdc_subset.c
@@ -218,7 +218,7 @@ static const struct driver_info	blob_inf
 /*-------------------------------------------------------------------------*/
 
 #ifndef	HAVE_HARDWARE
-#error You need to configure some hardware for this driver
+#warning You need to configure some hardware for this driver
 #endif
 
 /*
